NanoAnalysis/python/modules/jetJERC.py

```diff
@@ -73,7 +73,21 @@ def getJetCorrected(era, tag, is_mc, overwritePt=True) :
                 JERsfKey = None
             else:
                 raise ValueError("getJetCorrected: tag", era, "not supported")
-    elif era == 2023:
+    elif era == 2024:
+        if is_mc :
+            print("MC")
+        else :
+            folderKey = "2024_Winter24"
+            L1Key = "Winter24Prompt24_V3_DATA_L1FastJet_AK4PFPuppi"
+            L2Key = "Winter24Prompt24_V3_DATA_L2Relative_AK4PFPuppi"
+            L3Key = "Winter24Prompt24_V3_DATA_L3Absolute_AK4PFPuppi"
+            L2L3Key = "Winter24Prompt24_V3_DATA_L2L3Residual_AK4PFPuppi"
+            scaleTotalKey = None
+            smearKey = None
+            JERKey = None
+            JERsfKey = None
+
+    elif era == 2023 or era == 2025:
```

Comment:
Is there a specific reason to use 2023postBPIX corrections for 2025, instead of the 2024 ones? 
In any case, I would put a warning message here as well (see above)